telemetry/telemetry.go

```diff
@@ -68,36 +58,28 @@ type BridgeInfo struct {
 	ErrorMessage string
 }
 
-// Orchestrator Details structure.
-type OrchestratorInfo struct {
-	OrchestratorName    string
-	OrchestratorVersion string
-	ErrorMessage        string
-}
-
 // Azure CNI Telemetry Report structure.
 type CNIReport struct {
-	IsNewInstance       bool
-	CniSucceeded        bool
-	Name                string
-	Version             string
-	ErrorMessage        string
-	EventMessage        string
-	OperationType       string
-	OperationDuration   int
-	Context             string
-	SubContext          string
-	VMUptime            string
-	Timestamp           string
-	ContainerName       string
-	InfraVnetID         string
-	VnetAddressSpace    []string
-	OrchestratorDetails OrchestratorInfo
-	OSDetails           OSInfo
-	SystemDetails       SystemInfo
-	InterfaceDetails    InterfaceInfo
-	BridgeDetails       BridgeInfo
-	Metadata            common.Metadata `json:"compute"`
+	IsNewInstance     bool
+	CniSucceeded      bool
+	Name              string
+	Version           string
+	ErrorMessage      string
+	EventMessage      string
+	OperationType     string
+	OperationDuration int
+	Context           string
+	SubContext        string
+	VMUptime          string
+	Timestamp         string
+	ContainerName     string
+	InfraVnetID       string
+	VnetAddressSpace  []string
+	OSDetails         OSInfo
```

Comment:
why remove orchestratordetails but not interfgacedetails or bridgedetails

Reply:
They are used in [network.go](https://github.com/Azure/azure-container-networking/blob/master/cni/network/network.go#L275)